### PR TITLE
Contact form validation

### PR DESCRIPTION
**Pull request by sireclifford**
- Used JS to validate email address before form submit
- email address needs to be lowercase

### CODE REVIEW

File: scripts/form.js [hunk 1]
```diff
@@ -1,21 +1,41 @@
...
+destopForm.addEventListener('submit', (e) => {
+  const email = userEmail.value.trim();
+  const convertedEmail = email.toUpperCase();
+  if (email === convertedEmail) {
+    showErrorMsg('desktop');
+    e.preventDefault();
+  } else {
+    showSuccessMsg();
+  }
+});
```
Comment (KDlamini) [inline scripts/form.js]: When I submit the form with the email field containing uppercase characters nothing happens. Kindly look into this and make sure that:

- The error message should appear close to the "submit" button.
- The form should not be sent even if there's only one uppercase character on the email address. Currently, when I submit the form with the email having the first character as uppercase e.g `Example@email.com` the form is still submitted. This should not happen.

Why don't you try to check if the email is all lowercase by comparing if the input email and its lowercase converted version are the same instead:
```
email === email.toLowerCase()
```

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @sireclifford 🙋‍♀️,  

Good job so far! 💯 
There are some issues that you still need to work on improving your project according to the requirements. You are almost there! 💪🏾 

soclose

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: scripts/form.js (hunk 1 above)
Comment (sireclifford) [reply scripts/form.js]: @KDlamini that was a logical error right there. `.toLowerCase()` does a strict check. thank you.

**Review by koachcode — APPROVED** [review]
Hi @sireclifford , 

# STATUS: APPROVED 🟢

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**